README.md

```diff
@@ -6,9 +6,14 @@ This here is a twitter bot I wrote!
 but this isn't gonna stay a CLI forever. Mostly just testing. For now,
 the way you do this is:
 ```
-./update_status.py --secrets ~/path/to/secrets/file.yml --text "here is a tweet"
+PYTHONPATH=. python utils/update_status.py --secrets ~/path/to/secrets/file.yml --text "here is a tweet"
+```
+
+2. we can check to see if we created a new blogpost and then
+let people know about it!
+```
+PYTHONPATH=. python utils/blogpost_checker.py --secrets ~/path/to/secrets/file.yml
```

Comment:
`PYTHONPATH` malarkey is because at some point these utils will be called by some main entrypoint (probably).